src/lib/query.ts

```diff
@@ -24,5 +25,14 @@ export async function write(
   this: Connection,
   query: string
 ): Promise<WriteQueryResult> {
-  return await tablelandCalls.write.call(this, query);
+  if (this.options.rpcRelay) {
+    return await tablelandCalls.write.call(this, query);
+  }
+
+  // ask the Validator if this query is valid, and get the tableId for use in SC call
+  const { tableId } = await tablelandCalls.validateWriteQuery.call(this, query);
```

Comment:
If I'm specifying rpcRelay false here, am I going to be surprised that I still end up making calls to the remote host in this case?

Reply:
Maybe having an additional option with `validate: boolean` that defaults to `true`?

Reply:
how about `skipQueryValidation`? imo, flags with default value `true` are confusing.